### DISCUSSION TITLE
block / pressure pad / welcome mat that can modify backpack upgrades

### DISCUSSION THREAD

**Discussion by Edwin-Baranov** [P3pp3rF1y/SophisticatedBackpacks#267] [Ideas]
I am playing Enigmatica 6 right now and I love sophisticated backpacks however one issue that I've had was that most of the time I forget to turn off my pickup upgrade and have to go searching for my dropped items within my backpack while I'm in my base.

My Idea would be to have a special block (would love it if it was more like a carpet/mat) that has internal settings that when a player walks over it it would fire off an event that would turn off/on/toggle the specified upgrades. With this you can put the block at the entrance of your base, walk over it and not have to worry about auto pickups. To auto turn it back on you have another as an exit block outside your entrance.

Another idea would be a range block that has a specific radius with an OnEnter event and an OnExit event. This would be useful for the magnet upgrade similar to Botania's Solegnolia

**Comment by P3pp3rF1y** [comment]
Just on the magnet and Solegnolia - that exact thing is supported so magnet will get turned off by Solegnolia. Also Demagnetize can be used to stop magnet from picking up items in a range around demagnetize block.
